proto/attest.proto

```diff
@@ -250,9 +257,35 @@ message PlatformPolicy {
   GCEConfidentialTechnology minimum_technology = 3;
 }
 
+// A policy about what parts of a RIM to compare against machine state as
+// reflected in a quote or (verified) event log. Reference measurements for
+// a component are expected to be addressable by the machine state's reported
+// digest, or otherwise presented as cached collateral with the attestation
+// itself. The method of delivery is vendor-specific.
+message RIMPolicy {
+  // If true, the signed measurement must be available and the target
+  // measurement must be among the listed signed measurements.
+  // If false, then only error if there is a problem verifying the signed
+  // measurements when they are available.
+  bool require = 1;
+  // x.509 certificates in ASN.1 DER format.
+  repeated bytes root_certs = 2;
+}
+
+// Represent minimal decisions about attestation fields until necessary to
+// add for policy reasons.
+message SevSnpPolicy {
+  // The policy for checking the signed reference values for the UEFI at launch.
+  RIMPolicy uefi = 1;
```

Comment:
Is it possible to make this policy not just specific to snp?

I wonder later if tdx can have a similar firmware policy check